packages/http/src/Adaptor.js

```diff
@@ -172,6 +187,8 @@ export function del(path, params, callback) {
  * @param {String} body - data string to be parsed
  * @param {function} script - script for extracting data
  * @param {function} callback - (Optional) Callback function
+ * @state data - the parsed XML as a JSON object
+ * @state references - an array of all previous data objects used in the Job
```

Comment:
I've had to copy and paste references here. One thing I can't do is reference a type and then REMOVE a key.

But it's fine like this.

Another way to approach the problem would be to have a `BaseState` of `data`, `references` and `HTTPState` which adds `response`, and use BOTH as mixins. But it's not worth the overhead.